common/serial/Makefile

```diff
@@ -0,0 +1,73 @@
+#
+# Bareflank Hypervisor
+#
+# Copyright (C) 2015 Assured Information Security, Inc.
+# Author: Rian Quinn        <quinnr@ainfosec.com>
+# Author: Brendan Kerrigan  <kerriganb@ainfosec.com>
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this library; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
+
+################################################################################
+# FLAGS
+################################################################################
+
+CROSS_CC=~/opt/cross/bin/x86_64-elf-gcc
+CROSS_CXX=~/opt/cross/bin/x86_64-elf-g++
+CROSS_LD=~/opt/cross/bin/x86_64-elf-ld
+
+# CC=$(CROSS_CC)
+# CXX=$(CROSS_CXX)
+# LD=$(CROSS_LD)
+# RM=rm -rf
+# MD=mkdir -p
+
+CC=gcc
+CXX=g++
+LD=g++
+RM=rm -rf
+MD=mkdir -p
+
+CCFLAGS=
+CXXFLAGS=
+LDFLAGS=
+
+
+DEFINES=
+
+################################################################################
+# Sources
+################################################################################
+
+TARGET_NAME=bf_serial
+TARGET_TYPE=lib
+TARGET_CROSS_COMPILED=false
+
+OBJDIR=.build
+OUTDIR=../bin
+
+SOURCES=serial.cpp arch/arch_linux.cpp
```

Comment:
Nope that didn't fly for me. I got around it by manually touching the file. We should figure out what's necessary to support this scenario, as it's fairly common.